about.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="./styles/styles.css" />
+  </head>
+
+<body class="about-body">
+  <nav class="top-right open nav-container">
+    <a class="disc l1" href="index.html">Home</a>
+     
+    <a class="disc l2" href="about.html">About</a>
+
+    <a class="disc l3" href="portfolio.html">Portfolio</a>
+    
+    <a class="disc l4" href="hobby-blog.html">Hobbies</a>
+  </nav>
+  <main id="home">
+    <header>
+     <h1 class="about">about.</h1>
+     <h2 class="sub-title">I'm a current software development student at 
+    Ada Developers Academy in sometimes sunny Seattle, WA.</h2>
+    </header>
+      <p>
+        My name is Brittany (she/her) and I'm a current software
+        development student at Ada Developers Academy in Seattle, WA. Before
+        delving into coding I was a Patient Service Specialist at Harborview
+        Medical Center. While working at a hospital I challenged myself by
+        learning medical terminology. This was exciting because it gave me the
+        chance to be more efficient at communicating with medical professionals.
+        Work experience taught me that learning new skills and problem solving
+        regularly is what I enjoy. When I discovered programming I was excited
+        again - I could be constantly challenged while making a positive impact
+        to my clients' business.
+        
+        I am detail-oriented and analytical from my medical background, and from
+        my patient service specialist position I have gained immense empathy for
+        the people around me and stellar communication skills! I'm looking
+        forward to putting all of these skills together. I am extremely excited
+        to begin an internship as a software developer. I cannot wait to
+        exercise all the skills I've gained at Ada as well as learn new skills
+        that will take me to a full-time position. During and after my
+        internship, I hope to continue learning and to support other women and
+        gender-diverse people in tech.
+      </p>
```

Comment:
When we've got a break between paragraphs like this. Think about making them two `p` elements. That will then  give you the line break in between because of default styling already built into the `p` element.